src/test/java/com/google/cloud/spanner/pgadapter/JdbcMockServerTest.java

```diff
@@ -2101,6 +2102,60 @@ public void testSetToEmpty() throws SQLException {
     }
   }
 
+  @Test
+  public void testSetTimeZone() throws SQLException {
+    try (Connection connection = DriverManager.getConnection(createUrl())) {
+      connection.createStatement().execute("set time zone 'IST'");
+      verifySettingValue(connection, "timezone", "IST");
+    }
+  }
+
+  @Test
+  public void testSetTimeZoneToDefault() throws SQLException {
+    try (Connection connection = DriverManager.getConnection(createUrl())) {
```

Comment:
If you were to change this to the following:

```suggestion
    try (Connection connection = DriverManager.getConnection(createUrl()
                + "?options=-c%20timezone=IST")) {
```

The the `set time zone default` should set the time zone to `IST` instead of `UTC`.